bundles/org.openhab.binding.somfytahoma/src/main/java/org/openhab/binding/somfytahoma/internal/handler/SomfyTahomaBridgeHandler.java

```diff
@@ -563,7 +563,21 @@ public void sendCommand(String io, String command, String params, String url) {
 
         Boolean result = sendCommandInternal(io, command, params, url);
         if (!result) {
-            sendCommandInternal(io, command, params, url);
+            scheduler.schedule(() -> {
+                repeatSendCommandInternal(io, command, params, url, thingConfig.getRetries());
+            }, thingConfig.getRetryDelay(), TimeUnit.MILLISECONDS);
+        }
+    }
+
+    private void repeatSendCommandInternal(String io, String command, String params, String url, int retries) {
+        logger.info("Retrying command, retries left: {}", retries);
```

Comment:
There is no need to bother users with these messages. Debug severity is sufficient

Reply:
Done